src/ja/worker/proxy/proxy.py

```diff
@@ -10,28 +10,30 @@ class IWorkerServerProxy(ContinuousProxy, ABC):
     """
     Interface for the proxy for the central server used on the worker client.
     """
-    def __init__(self, ssh_config: SSHConfig):
+    def __init__(self, uid: str, ssh_config: SSHConfig):
```

Comment:
why do we need the uid here

Reply:
the commands are called from the worker main, what is the point of having this uid if you don't use it in the commands?

Reply:
I think that was a mistake. I honestly don't remember why i put it there.